monitoring/prometheus/prometheus-configs.yaml

```
@@ -290,35 +290,59 @@ data:
+        - alert: RHODS JupyterHub Probe Success Burn Rate
           annotations:
             message: 'High error budget burn for {{ $labels.instance }} (current value: {{ $value }}).'
             triage: "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/JupyterHub/rhods-probe-success-burn-rate.md"
-            summary: RHODS Route Error Burn Rate
+            summary: RHODS JupyterHub Probe Success Burn Rate
           expr: |
-            sum(probe_success:burnrate5m{instance=~"jupyterhub|rhods-dashboard|jupyterhub-db|traefik"}) by (instance) > (14.40 * (1-0.99950))
+            sum(probe_success:burnrate5m{instance=~"jupyterhub|jupyterhub-db"}) by (instance) > (14.40 * (1-0.99950))
             and
-            sum(probe_success:burnrate1h{instance=~"jupyterhub|rhods-dashboard|jupyterhub-db|traefik"}) by (instance) > (14.40 * (1-0.99950))
+            sum(probe_success:burnrate1h{instance=~"jupyterhub|jupyterhub-db"}) by (instance) > (14.40 * (1-0.99950))
           for: 2m
           labels:
             severity: critical
-        - alert: RHODS Probe Success Burn Rate
+        - alert: RHODS JupyterHub Probe Success Burn Rate
           annotations:
             message: 'High error budget burn for {{ $labels.instance }} (current value: {{ $value }}).'
             triage: "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/JupyterHub/rhods-probe-success-burn-rate.md"
-            summary: RHODS Route Error Burn Rate
+            summary: RHODS JupyterHub Probe Success Burn Rate
           expr: |
-            sum(probe_success:burnrate30m{instance=~"jupyterhub|rhods-dashboard|jupyterhub-db|traefik"}) by (instance) > (6.00 * (1-0.99950))
+            sum(probe_success:burnrate30m{instance=~"jupyterhub|jupyterhub-db"}) by (instance) > (6.00 * (1-0.99950))
             and
-            sum(probe_success:burnrate6h{instance=~"jupyterhub|rhods-dashboard|jupyterhub-db|traefik"}) by (instance) > (6.00 * (1-0.99950))
+            sum(probe_success:burnrate6h{instance=~"jupyterhub|jupyterhub-db"}) by (instance) > (6.00 * (1-0.99950))
           for: 15m
           labels:
             severity: critical
         - alert: RHODS Probe Success Burn Rate
           annotations:
             message: 'High error budget burn for {{ $labels.instance }} (current value: {{ $value }}).'
             triage: "https://gitlab.cee.redhat.com/service/managed-tenants-sops/-/blob/main/RHODS/JupyterHub/rhods-probe-success-burn-rate.md"
-            summary: RHODS Route Error Burn Rate
+            summary: RHODS Probe Success Burn Rate
+          expr: |
+            sum(probe_success:burnrate5m{instance=~"rhods-dashboard|traefik"}) by (instance) > (14.40 * (1-0.99950))
+            and
+            sum(probe_success:burnrate1h{instance=~"rhods-dashboard|traefik"}) by (instance) > (14.40 * (1-0.99950))
+          for: 2m
+          labels:
+            severity: warning
```

```suggestion
            severity: critical
```
I think we may want to leave this as `critical`. From Pagerduty I see only 3 alerts generated from this highest degree of burnrate for SRE and only 1 of those auto-resolved as an anomaly. We can reduce this to `warning` later, but at that point we may have to re-evaluate the SLO itself if we are no longer concerned about error budget.

---

We moved this to `warning` because we have a `Route Burn Rate Alert` for Dashboard as well that is `critical`

---

Here is the alert : https://github.com/VaishnaviHire/odh-deployer/blob/rhods_4071/monitoring/prometheus/prometheus-configs.yaml#L230

---

Cool, then `warning` here is okay with me.